products/jse/features/jse_editor.feature

```diff
@@ -3,9 +3,11 @@ Feature: Editor
   As an Editor
   I want to check for my ability to complete my tasks
 
+  Background:
```

Comment:
This background would require all scenarios to use the @api tag in this feature. Take the tag of of each individual scenario and place the @api tag as the very first line in this file.